src/rust/scheduler/scheduler.rs

```diff
@@ -402,39 +421,82 @@ mod tests {
         Ok(())
     }
 
-    /// Tests if consecutive tasks are not assigned the same task id.
     #[test]
-    fn test_scheduler_task_ids() -> Result<()> {
+    fn from_task_id_returns_none_for_non_existing_task_id() -> Result<()> {
         let scheduler: Scheduler = Scheduler::default();
+        match scheduler.from_task_id(0) {
+            Some(_) => anyhow::bail!("from_task_id() must return None"),
+            None => {},
+        }
+        Ok(())
+    }
 
-        // Create and run a task.
-        let task: DummyTask = DummyTask::new(String::from("testing"), Box::pin(DummyCoroutine::new(0)));
+    #[test]
+    fn from_task_id_returns_correct_task_handle() -> Result<()> {
```

Comment:
Could you add a Rust Doc comment for this test, like we have for other tests in this module?